src/main/scala/org/apache/spark/sql/delta/DeltaConfig.scala

```diff
@@ -377,7 +377,14 @@ object DeltaConfigs extends DeltaLogging {
     "needs to be larger than or equal to -1.")
 
   val SYMLINK_FORMAT_MANIFEST_ENABLED = buildConfig[Boolean](
-    s"${hooks.GenerateSymlinkManifest.CONFIG_NAME_ROOT}.enabled",
+    s"${hooks.SymlinkGenerateManifest.CONFIG_NAME_ROOT}.enabled",
```

Comment:
Lets not change the config here, we can just keep hooks.`GenerateSymlinkManifest`